Demo/Application/Features/Venmo - Custom Button/BraintreeDemoCustomVenmoButtonViewController.m

```diff
@@ -36,7 +36,7 @@ - (void)tappedCustomVenmo {
         } else if (error) {
             self.progressBlock(error.localizedDescription);
         } else {
-            self.progressBlock(@"Canceled 🔰");
+            self.progressBlock(@"Cancelled 🔰");
```

Comment:
Is there a reason this changed? Our style guide uses Canceled (with one `L`) https://internal.braintreepayments.com/display/ID/Style+Guide